origin.html

```diff
@@ -91,6 +91,27 @@ <h2 class="aboutmetitle">About me</h2>
                 <li class="subline">Terminal</li>
             </ul>
         </div>
+            
+        <div class="informationsection">
+            <form action="https://formspree.io/f/mbjwogoo" method="get">
+                <h2 class="informationtitle">I'm always interested in hearing about new proijects, so if you'd like to chant plese get in</h2>
+                <input type="text" placeholder="Full Name" maxlength="30" required>
+                <p class="undeline2"><img class="underline2" src="./images/information/Container.png" alt=""></p>
+                <input type="email" placeholder="Email address" required>
+                <p class="undeline2"><img class="underline2" src="./images/information/Container.png" alt=""></p>
+                <textarea class="textareainformation" name="enter-text-here" maxlength="500" rows="10" placeholder="Add your text" required></textarea>
+                <button type="submit" class="informationbutton">Get in touch</button>
```

Comment:
Please ensure that button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Also please ensure you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).